index.css

```diff
@@ -158,6 +169,134 @@
 
 /* -------------------------------------------------------------------------------------------------------- */
 
+.Popup-wrapper {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  backdrop-filter: blur(5px);
+}
+
+.Popup {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  color: #344563;
+  font-family: 'Inter', sans-serif;
+  height: 85vh;
+  width: 85vw;
+  background: #fff;
+  padding: 16px;
+}
```

Comment:
- [ ] You did great work until now, just one thing here is you have to make the project pic on the left and the description and the buttons on the right to match the Figma design.
To solve it you can wrap them in a container and use a display flex to make them next to each other.

Your design
![image](https://user-images.githubusercontent.com/35535130/161953040-181f3d26-e338-4e40-bead-12d9cc3513de.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/161953154-74660c9a-0f4e-4323-a6cc-04d8649d7227.png)